codalab/worker/local_run/docker_image_manager.py

```diff
@@ -91,8 +92,9 @@ def _cleanup(self):
             but because of (1) we don't want to use that.
         """
         while not self._stop:
+            # We sort the image cache in LRU order
             deletable_entries = set(self._image_cache.values())
-            disk_use = sum(cache_entry.virtual_size for cache_entry in deletable_entries)
+            disk_use = sum(entry.virtual_size for entry in deletable_entries)
```

Comment:
I changed the variable name here just for consistency. At the bottom of this function, there is a similar logic that uses `entry`.